mod/qbank/db/install.php

```diff
@@ -23,12 +23,26 @@
  * @license     https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+use mod_qbank\helper;
+
 defined('MOODLE_INTERNAL') || die();
 
 /**
  * Custom code to be run on installing the plugin.
  */
 function xmldb_qbank_install() {
 
-    return true;
+    // Get data from populated questions.
+    $questioncategories = helper::get_categories_populated();
+    foreach ($questioncategories as $qcategory) {
+        $contextlevel = (int)$qcategory->contextlevel;
+
+        if ($contextlevel === CONTEXT_COURSECAT) {
+            $coursename = substr("qbank_" . CONTEXT_COURSECAT . $qcategory->name, 0, 254);
```

Comment:
I know I gave zero direction here, But we should come up with a more human friendly course name.
Maybe "Question bank: . $qcategory->name" still with the trimming

Reply:
Modified `$coursename` as asked